app/assets/javascripts/dragdrop.js

```diff
@@ -1,4 +1,4 @@
-jQuery(document).ready(function() {
```

Comment:
Does the new syntax also works like before? We want this script to work after document is loaded.

Reply:
Oh yes, this script is still working. I changed the syntax because I believe the old one was deprecated.